src/Edm/Validation/Internal/InterfaceValidator/VisitorOfIBinaryValue.php

```diff
@@ -10,10 +10,10 @@
 
 class VisitorOfIBinaryValue extends VisitorOfT
 {
-    protected function VisitT($value, array &$followup, array &$references): iterable
+    protected function VisitT($value, array &$followup, array &$references): ?iterable
     {
         assert($value instanceof IBinaryValue);
-        return $value->getValue() == null ?
+        return null !== $value->getValue() ?
             [InterfaceValidator::CreatePropertyMustNotBeNullError($value, 'Value') ]
```

Comment:
@c-harris, this seems to be only place in codebase where you're letting IBinaryValue->getValue() be nullable.  EdmValueWriter::BinaryAsXml, EdmModelCsdlSchemaWriter->writeRequiredAttribute and ExpressionTypeChecker::TryAssertBinaryConstantAsType all assume it's not nullable.

I've spotted the sense-flip in my previous cleanup, but that's small potatoes compared to the contradiction of getValue()'s nullability.